utils/i2csfp/Makefile

```
@@ -0,0 +1,39 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=i2csfp
+PKG_VERSION:=20240616
+PKG_RELEASE:=1
+PKG_LICENSE:=GPL-3.0
+
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_VERSION:=ca8e40767e4e14d9e133f73f4689b74baaa0aeaa
+PKG_SOURCE_URL:=https://github.com/ericwoud/i2csfp
+PKG_MIRROR_HASH:=329d9c72b8e4369bf053c281144637291d3ace04658efe4778a1adda7eac54d6
+
+include $(INCLUDE_DIR)/package.mk
+
+define Package/i2csfp
+  SECTION:=utils
+  CATEGORY:=Utilities
+  TITLE:=I2C SFP utility
+  MAINTAINER:=Daniel Golle <daniel@makrotopia.org>
+endef
+
+define Package/i2csfp/description
+   A user-space program to access sfp module via i2c.
```

```suggestion
  A user-space program to access sfp module via i2c.
```

use 2 spaces for indentation